grafanalib/zabbix.py

```diff
@@ -655,3 +692,164 @@ def zabbixTextTarget(application, group, host, item, text,
+        with different color
+    :param minSpan: defines panel minimum spans
+    :param okEventColor: defines color for triggers with Ok status
+    :param pageSize: defines number of triggers per panel page
+    :param scroll: defines if scroll should be shown
+    :param severityField: defines if severity field should be shown
+    :param showEvents: defines event type to query (Ok, Problems, All)
+    :param showTriggers: defines trigger type to query
+        (all, acknowledged, unacknowledged)
+    :param sortTriggersBy: defines trigger sort policy
+    :param span: defines span number for panel
+    :param statusField: defines if status field should be shown
+    :param transparent: defines if panel should be transparent
+    :param triggerSeverity: defines colors for trigger severity
+    :param triggers: trigger query
+
+    """
+    dataSource = attr.ib()
+    title = attr.ib()
+
+    ackEventColor = attr.ib(default=BLANK,
+                            validator=instance_of(RGBA))
+    ageField = attr.ib(default=True, validator=instance_of(bool))
+    customLastChangeFormat = attr.ib(default=False,
+                                     validator=instance_of(bool))
+    description = attr.ib(default="", validator=instance_of(str))
+    fontSize = attr.ib(default=Percent(100), validator=instance_of(Percent))
+    height = attr.ib(default=DEFAULT_ROW_HEIGHT, validator=instance_of(Pixels))
+    hideHostsInMaintenance = attr.ib(default=False,
+                                     validator=instance_of(bool))
+    hostField = attr.ib(default=True, validator=instance_of(bool))
+    hostTechNameField = attr.ib(default=False, validator=instance_of(bool))
+    id = attr.ib(default=None)
+    infoField = attr.ib(default=True, validator=instance_of(bool))
+    lastChangeField = attr.ib(default=True, validator=instance_of(bool))
+
+    lastChangeFormat = attr.ib(default="")
+    limit = attr.ib(default=10, validator=instance_of(int))
+    links = attr.ib(default=attr.Factory(list),
+                    validator=is_list_of(DashboardLink))
+    markAckEvents = attr.ib(default=False, validator=instance_of(bool))
+    minSpan = attr.ib(default=None)
+    okEventColor = attr.ib(default=GREEN,
+                           validator=instance_of(RGBA))
+    pageSize = attr.ib(default=10, validator=instance_of(int))
+    repeat = attr.ib(default=None)
+    scroll = attr.ib(default=True, validator=instance_of(bool))
+    severityField = attr.ib(default=False, validator=instance_of(bool))
+    showEvents = attr.ib(default=ZABBIX_EVENT_PROBLEMS)
+    showTriggers = attr.ib(default=ZABBIX_TRIGGERS_SHOW_ALL)
+    sortTriggersBy = attr.ib(default=ZABBIX_SORT_TRIGGERS_BY_CHANGE)
+    span = attr.ib(default=None)
+    statusField = attr.ib(default=False, validator=instance_of(bool))
+    transparent = attr.ib(default=False, validator=instance_of(bool))
+    triggerSeverity = attr.ib(
+        default=createZabbixSeverityColors(ZABBIX_SEVERITY_COLORS))
```

Comment:
This also needs to be defined with `attr.Factory`, otherwise all the `triggerSeverity` will be shared across all of the instances that use the default.